book.rb

```diff
@@ -0,0 +1,17 @@
+require_relative './rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.book = self
+  end
```

Comment:
- [ Required ] I would like to suggest that you refactor this method to take a `person` , `date` as arguments and then utilize the `Rental` class constructor to create a rental, this will ensure that `Person` is set in `Rentals` constructor. You can do this by refactoring your code as shown below.

```ruby
   def add_rental(person, date)
      Rental.new(date, self, person)
   end
```